templates/resolv.conf.tmpl

```diff
@@ -23,8 +23,12 @@ sortlist {% for sort in sortlist %}{{sort}} {% endfor %}
 {% endif %}
 {% if options or flags %}
 
-options {% for flag in flags %}{{flag}} {% endfor %}
-{% for key, value in options.items() -%}
- {{key}}:{{value}}
+options
```

Comment:
It was not immediately obvious to me (as a developer/user on a distro that doesn't use resolvconf) that the intended behaviour here is for all the flags and options to be on the same line. Could we possibly add a `{# comment #}` to note that, so future readers have an easier time grokking what's happening here?

Reply:
Hi,
Just pushed the requested change.
Please let me know if it works for you.

Thanks!